community/socklog/socklog.setup

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+NORMAL="\033[1;0m"
+STRONG="\033[1;1m"
+RED="\033[1;31m"
+GREEN="\033[1;32m"
+runscript=/etc/sv/socklog-unix/run
+
+print_stop() {
+	local prompt="${STRONG}$1 ${RED}$2${NORMAL}"
+        printf "${prompt} %s"
+}
+
+print_start() {
+	local prompt="${STRONG}$1 ${GREEN}$2${NORMAL}"
+        printf "${prompt} %s"
+}
+
+print_stop "\nSTOP BusyBox Syslog & remove it from the Boot Runlevel ?" "[ Enter or Y to remove ]:"; read ans
+if [ -z "$ans" ] || [ "$ans" = "y" ] || [ "$ans" = "Y" ]; then
+        echo; rc-update del syslog boot; rc-service syslog stop; rc-service klogd stop
+fi
+
+print_start "\nSTART & enable socklog with runit ?" "[ Enter or Y to add ]:"; read ans
+if [ -z "$ans" ] || [ "$ans" = "y" ] || [ "$ans" = "Y" ]; then
+        if [ -f $runscript ] && [ ! -x $runscript ]; then
+                echo -e "\nenabling: $runscript"
+                chmod 700 $runscript
+	fi
+
+	if ! grep -q runit /etc/apk/world; then
```

Comment:
This isn’t good method how to check if runit is installed. What if it’s installed as a dependency of other package? It’d be better to check if some runit binary exists.